### ISSUE TITLE
-Wunused in kernel/sysctl.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#407]
KernelCI reports:
```
    3    kernel/sysctl.c:3331:12: warning: unused function 'proc_dointvec_minmax_bpf_stats' [-Wunused-function]
```

**Comment by nathanchance** [comment]
This has been fixed already in the BPF tree: https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git/commit/?id=3fcc5530bcb2a879e32bd940e6eafee328ac3647

**Comment by kragniz** [comment]
@nathanchance how do you find fixes in various trees? Do you just keep an eye on them manually, or is there some better way?

**Comment by nathanchance** [comment]
I'm subscribed to the linux-next mailing list where a lot of this gets reported by Stephen Rothwell using GCC and the maintainers usually respond that they are working on a fix or whatever. Additionally, I search https://lore.kernel.org/lkml/ for key words to see if any of the other common warning fixers have sent a patch and the maintainer's tree to see if it has been fixed there.

**Comment by kragniz** [comment]
nice, thanks

**Comment by nathanchance** [comment]
This was fixed in `next-20190301`: https://git.kernel.org/next/linux-next/c/3fcc5530bcb2a879e32bd940e6eafee328ac3647
